locales/ru_RU/pcsx2_Main.po

```diff
@@ -2246,11 +2267,20 @@ msgstr ""
 "Предварительная загрузка хака TLB во избежание пропуска TLB в игре Goemon"
 
 #: pcsx2/gui/Panels/GameFixesPanel.cpp:108
+msgid "VU I bit Hack avoid constant recompilation. Fixes memory leak slowdowns/stalls."
+msgstr ""
+"VU I bit Hack во избежание постоянной рекомпиляции. Исправляет "
+"тормоза/зависания из-за утечек памяти."
+
+#: pcsx2/gui/Panels/GameFixesPanel.cpp:109
 msgid ""
-"VU I bit Hack avoid constant recompilation (Scarface The World Is Yours)"
+"Try this when \"microVU1: Program cache limit\" is printed in the log.\n"
+"Known to affect following games:\n * Scarface The World Is Yours\n * "
+"Crash Tag Team Racing"
```

Comment:
Looks like I forgot to revert the change when I removed the gamefix a few days ago so you don't need to include Crash Tag Team Racing.